App/Component/News/Posts/index.js

```diff
@@ -13,45 +14,84 @@ import {
   AutoExpandingTextInput,
 } from '~/Component';
 
-const defaultAvatar = Images.avatar['male08'];
+const ImagePickerConfig = {
+  allowsEditing: true,
+  aspect: [4, 3],
```

Comment:
Why a news photo must have aspect of 3:4 ?

Reply:
Because we should use the same resolution for every picture upload to server